.binder/requirements.txt

```
@@ -0,0 +1,10 @@
+bqplot
+ipyleaflet
+jupyterlab>=2
```

We can probably bump to `jupyterlab>=3` now:

```suggestion
jupyterlab~=3.0
```